sdk/quantum/service.projects

```diff
@@ -0,0 +1,25 @@
+
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - "all" will build all projects
+   - "client" will build the new track 2 client/data libraries
+   - "mgmtclient" will build the track 2 mgmt libraries
+   - "data" will build the track 1 data plane libraries
```

Comment:
Do we still support those?

Reply:
We do support client and mgmtclient. I just copied this file as other SDKs have them like that, but probably we can remove the legacy "data" track1 client, but I wasn't sure...